client/web/src/settings/ApplicationComponent.js

```diff
@@ -107,20 +112,21 @@ export function ApplicationComponent(props) {
   const filesystem = {
     ...appConfig.filesystem,
     mountPoint: appConfig.filesystem?.mountPoint || '',
-    // Somehow the container OS was set without the filesystem type being set
-    // There's a handler on the OS radio so it'll overwrite this as necessary, but start off
-    // with FSX if Windows and EFS if Linux
-    fileSystemType: appConfig.filesystem?.fileSystemType || os !== 'LINUX' ? 'FSX' : 'EFS',
+    // Start off with FSX if Windows and EFS if Linux
+    fileSystemType: appConfig.filesystem?.fileSystemType || (os !== 'LINUX' ? 'FSX' : 'EFS'),
```

Comment:
This wasn't working because I suspect the `||` had higher priority than the ternary so it was an order of operations issue. Thanks to @brtrvn for the tip.